mythtv/bindings/python/MythTV/__init__.py

```diff
@@ -42,6 +43,6 @@
 from .methodheap import *
 
 
-__version__ = OWN_VERSION
+__version__ = ".".join(str(x) for x in _versions.OWN_VERSION)
```

Comment:
MYTHTV_PYTHON_OWN_VERSION is unique to the python bindings. If it doesn't have to be a list in _versions.py, you could change configure to write the string '34.0.-1.0' and not have to manipulate it here.

Reply:
The defined `OWN_VERSION` is kept for legacy reasons. I've seen some script that uses it.